src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -0,0 +1,71 @@
+module sonic-feature{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-feature";
+    prefix port;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "Feature Table yang Module for SONiC OS";
+
+    container sonic-feature {
+
+        container FEATURE {
+
+            description "feature table in config_db.json";
+
+            list FEATURE_LIST {
+
+                key "name";
+
+                leaf name {
+                    description "feature name in Feature table";
+                    type string {
+                        length 1..32;
+                    }
+                }
+
+                leaf state {
+                    description "state of the feature";
+                    type stypes:feature_state;
+                }
+
+                leaf auto_restart {
+                    type stypes:feature_state;
+
+                }
+
+                leaf has_timer {
+                    description "This configuration identicates if there is
+                                timer associated to this feature";
+                    type boolean;
+                    default false;
+                }
+
+                leaf has_global_scope {
+                    description "This configuration identicates there will only one service
+                                spawned for the device";
+                    type boolean;
+                    default false;
+                }
+
+                leaf has_per_asic_scope {
+                    description "This configuration identicates there will only one service
+                                spawned per asic";
+                    type boolean;
+                    default false;
+                }
+
+                leaf high_mem_alert {
+                    description "This configuration controls the trigger to generate
+                                alert on high memory utilization";
+                    type stypes:feature_state;
+                    default "disabled";
+                }
```

Comment:
no field for high_cpu_alert?

Reply:
I remember, there was a file for CPU & memory allocation per feature/docker, is that not exposed to the user yet?

Reply:
currently there is no support for `high_cpu_alert `now. We update once the feature is supported